gpu-enabler/src/main/scala/com/ibm/gpuenabler/ColumnPartitionSchema.scala

```diff
@@ -252,15 +260,15 @@ private[gpuenabler] class ColumnSchema(
     out.writeObject(propertyChain)
   }
 
-  private def readObject(in: ObjectInputStream): Unit = CUDAUtils.sparkUtils.tryOrIOException {
+  private def readObject(in: ObjectInputStream): Unit = CUDAUtils.sparkUtils.tryOrIOException { 
     val mirror = ColumnPartitionSchema.mirror
     _columnType = in.readObject().asInstanceOf[ColumnType]
     _terms =
       in.readObject().asInstanceOf[Vector[(String, String)]].map { case (clsName, propName) =>
         val cls = CUDAUtils.sparkUtils.classForName(clsName)
         val typeSig = mirror.classSymbol(cls).typeSignature
-        typeSig.decl(universe.TermName(propName)).asTerm
-        // typeSig.declaration(universe.stringToTermName(propName)).asTerm // Scala_2.10
+        typeSig.declaration(universe.stringToTermName(propName)).asTerm
```

Comment:
Can we use scala 2.11 as default?

Reply:
No.
I want to revise this lines and use scala 2.10 as default.
I didn't confirm the whole code in order to modify scala version.